gbpservice/nfp/configurator/lib/demuxer.py

```diff
@@ -105,45 +103,56 @@ def get_service_agent_info(self, operation, service_type, request_data):
         """
 
         sa_info_list = []
+        vendor_map = {const.FIREWALL: const.VYOS,
```

Comment:
It can't be because this 'default' case was added in order to default the
requests to a particular driver even in the presence of 'n' number of
drivers. Once we mandate service_vendor, we can remove this entire vendor
map.

Regards,
Deepak

On Wed, Apr 27, 2016 at 2:04 PM, njagadish <notifications@github.com> wrote:

> In gbpservice/nfp/configurator/lib/demuxer.py
> <https://github.com/oneconvergence/group-based-policy/pull/251#discussion_r61221726>
> :
>
> > @@ -105,45 +103,56 @@ def get_service_agent_info(self, operation, service_type, request_data):
> >          """
> >
> >          sa_info_list = []
> > +        vendor_map = {const.FIREWALL: const.VYOS,
>
> Why is vendor map hard coded? Can't this be prepared base on what drivers
> got loaded?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/oneconvergence/group-based-policy/pull/251/files/9c68054f9caa237f1086b0c64846ae0c6080839c#r61221726>
>